components/ca-form.js

```diff
@@ -333,8 +333,8 @@ define([
 
                 // erase current form
                 clearElement(this.form);
-                this.setAttribute('create-url', '');
-                this.setAttribute('update-url', '');
+                this.removeAttribute('create-url');
```

Comment:
Why are you touching the attributes directly? Why not use the createUrl and updateUrl setter that are provided to set and get the attributes...